src/Orleans.Runtime/GrainDirectory/LocalGrainDirectory.cs

```diff
@@ -484,9 +485,10 @@ public SiloAddress CalculateTargetSilo(GrainId grainId, bool excludeThisSiloIfSt
                     string grainName;
                     if (!Constants.TryGetSystemGrainName(grainId, out grainName))
                         grainName = "MembershipTableGrain";
-                    
-                    var errorMsg = grainName + " cannot run without Seed node - please check your silo configuration file and make sure it specifies a SeedNode element. " +
-                        " Alternatively, you may want to use AzureTable for LivenessType.";
+
+                    var errorMsg = $"{grainName} cannot run without Seed node. Please check your silo configuration make sure it specifies a SeedNode element. " +
```

Comment:
Seed node here is really Primary node, right? And there's at most one? The concept of having multiple was never implemented afaik

Reply:
Yes, I validated that there current be more than 1 seed node, and it's indeed the same as the primary. Right now it's really messed up, but we are getting rid of this concept. See https://github.com/dotnet/orleans/issues/3507